src/wazuh_mcp_server/utils/pydantic_compat.py

```
@@ -0,0 +1,132 @@
+    
+    else:
+        # Pydantic V1 - Use directly  
+        from pydantic import BaseModel, Field, validator
+        ValidationInfo = None
+        
+        # Log success for non-Fedora systems
+        if not PLATFORM_INFO['is_fedora']:
+            import logging
+            logging.info(f"Pydantic V1 detected on {PLATFORM_INFO['system']} - using native mode")
+
+except ImportError as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)